app/scripts/modules/core/src/pipeline/config/stages/managed/ImportDeliveryConfigExecutionDetails.tsx

```diff
@@ -2,10 +2,62 @@ import React from 'react';
 
 import { ExecutionDetailsSection, IExecutionDetailsSectionProps, StageFailureMessage } from 'core/pipeline';
 import { IGitTrigger } from 'core/domain';
+import { CollapsibleSection, Markdown } from 'core/presentation';
 import { SETTINGS } from 'core/config';
 
 const NOT_FOUND = 'Not found';
 
+interface IDeliveryConfigImportErrorDetails {
+  error: string;
+  message: string;
+  pathExpression: string;
+}
+
+interface IDeliveryConfigImportError {
+  message: string;
+  details?: IDeliveryConfigImportErrorDetails;
+}
+
+const CustomErrorMessage = (message: string, debugDetails?: string) => {
+  return (
+    <div>
+      <div className="alert alert-danger">
+        <b>There was an error parsing your delivery config file.</b>
+        <br />
+        <Markdown message={message} />
+        <br />
+        {debugDetails && (
+          <CollapsibleSection heading={({ chevron }) => <span>{chevron} Debug Details</span>}>
+            <pre>{debugDetails}</pre>
+          </CollapsibleSection>
+        )}
+      </div>
+    </div>
+  );
+};
+
+function buildCustomErrorMessage(error: IDeliveryConfigImportError) {
+  if (!error) {
+    return null;
+  }
+
+  if (!error.details) {
+    return CustomErrorMessage(error.message);
+  }
+
+  const pathExpression = error.details.pathExpression.replace('.', '/');
+  /* eslint-disable */
+  const errorMessage =
+    {
+      missing_property: `The following property is missing: \`${pathExpression}\``,
+      invalid_type: `The type of property \`${pathExpression}\` is invalid.`,
+      invalid_format: `The format of property \`${pathExpression}\` is invalid.`,
+      invalid_value: `The value of property \`${pathExpression}\` is invalid.`,
+    }[error.details.error] || 'Unknown error';
```

Comment:
The code formatter we run in the git pre-commit hook is changing the properties in this object from strings to symbols as you can see. This is what's causing the build to break:
```
  src/pipeline/config/stages/managed/ImportDeliveryConfigExecutionDetails.tsx:51:5 - error TS7053: Element implicitly has an 'any' type because expression of type 'string' can't be used to index type '{ missing_property: string; invalid_type: string; invalid_format: string; invalid_value: string; }'.
    No index signature with a parameter of type 'string' was found on type '{ missing_property: string; invalid_type: string; invalid_format: string; invalid_value: string; }'.
```
How do I stop the formatter from messing with my code? 🤔